GVFS/GVFS.Common/Prefetch/BlobPrefetcher.cs

```diff
@@ -83,70 +84,36 @@ public class BlobPrefetcher
 
         public List<string> FolderList { get; }
 
-        public static bool TryLoadFolderList(Enlistment enlistment, string foldersInput, string folderListFile, List<string> folderListOutput, out string error)
+        public static bool TryLoadFolderList(Enlistment enlistment, string foldersInput, string folderListFile, bool readListFromStdIn, List<string> folderListOutput, out string error)

```

Comment:
it can't be the last argument in the list, as the last argument is `out string error`, so I've moved it to be a penultimate argument. Also, that means, that `error:` needs to be passed by name as well. Both pushed.